content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions.  FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/).  Unfortunately, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.
+
+## Design
+
+There are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
```

Comment:
It looks like the preview isn't rendering these numbered bullet points. You may need to use bullet points throughout this entire document.

Reply:
I checked the HTML output, and it is correctly using \<ol\>...\</ol\> tags:
```html
 <p>There are two components to every <code>veneer</code>:</p>
<ol>
<li>An arbitrary API</li>
<li>An executable which processes #1 and produces a valid FBC.</li>
</ol>
```
so this looks like just a preview problem.